bin/cml/tensorboard/connect.js

```diff
@@ -28,8 +29,8 @@ const tbLink = async (opts = {}) => {
     chrono = chrono + chronoStep;
   }
 
-  const error = await fs.readFile(stderror, 'utf8');
-  throw new Error(`Tensorboard took too long. ${error}`);
+  winston.error(await fs.readFile(stderror, 'utf8'));
```

Comment:
This is also fixing TB error stacktrace telemetry. An issue that its not opened as an issue, just in discord.
I would either open an issue and close it here or change the PR to something that may include this?

Reply:
Updated the pull request body